packages/stats/gbstats/gbstats.py

```diff
@@ -231,40 +218,12 @@ def get_configured_test(
             stat_b, RegressionAdjustedStatistic
         ):
             raise ValueError(RA_NOT_COMPATIBLE_WITH_BAYESIAN_ERROR)
-        stat_a_proportion = isinstance(stat_a, ProportionStatistic)
-        stat_b_proportion = isinstance(stat_b, ProportionStatistic)
-        stat_a_quantile = isinstance(
-            stat_a, (QuantileStatistic, QuantileClusteredStatistic)
-        )
-        stat_b_quantile = isinstance(
-            stat_b, (QuantileStatistic, QuantileClusteredStatistic)
-        )
 
-        if stat_a_proportion and stat_b_proportion:
-            return BinomialBayesianABTest(
-                stat_a,
-                stat_b,
-                BinomialBayesianConfig(**base_config, inverse=metric.inverse),
-            )
-        elif stat_a_quantile and stat_b_quantile:
-            return GaussianEffectABTest(
-                stat_a,
-                stat_b,
-                EffectBayesianConfig(**base_config, inverse=metric.inverse),
-            )
-        elif (
-            not stat_a_proportion
-            and not stat_b_proportion
-            and not stat_a_quantile
-            and not stat_b_quantile
-        ):
-            return GaussianBayesianABTest(
-                stat_a,
-                stat_b,
-                GaussianBayesianConfig(**base_config, inverse=metric.inverse),
-            )
-        else:
-            raise ValueError(COMPARE_PROPORTION_NON_PROPORTION_ERROR)
+        return GaussianEffectABTest(
+            stat_a,
+            stat_b,
+            GaussianEffectBayesianConfig(**base_config, inverse=metric.inverse),
```

Comment:
I know GaussianEffectBayesianConfig permits prior mean specification.  Can you please confirm that customer will be able to specify prior mean in the UI?  In the PR it looked like they could specify only prior SD, but maybe I was missing something.

Reply:
Since updated as per our conversation.